src/ftxui/dom/border.cpp

```diff
@@ -12,10 +13,11 @@
 
 namespace ftxui {
 
-static wchar_t simple_border_charset[] = L"╭╮╰╯─│┬┴┤├";
+static std::string simple_border_charset[] = {"╭", "╮", "╰", "╯", "─",
+                                              "│", "┬", "┴", "┤", "├"};
```

Comment:
Consider using `Utf8ToGlyphs` function to split these glyphs from a single raw string literal.

Similarly for charset in gauge and graph.